### PR TITLE
Feat: Lesson17 (1. Create 5 image tags)

### PR DESCRIPTION
**Pull request by yuka830**
# [Lesson17](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#%E3%82%B9%E3%83%A9%E3%82%A4%E3%83%89%E3%82%B7%E3%83%A7%E3%83%BC%E4%BD%9C%E6%88%90)
```
仕様
1.画面遷移してから3秒後に解決されるPromiseが返すオブジェクトを元にimgタグを5つつくる。

2.それぞれは.z-indexで重ねた状態。矢印画像をクリックを押すとスライド画像が変わる

3.5枚中何枚目かを表示して、5/5の場合Nextの矢印はdisabledにする。1/5枚の時はBackボタンはdisabledにする
```
## [CodeSandbox](https://codesandbox.io/s/lesson17-create-5-image-tags-cipc1)
Last commit: [1915f5d](https://github.com/yuka830/js-lessons/pull/20/commits/1915f5dc898cc7bf9d413a70ca11cc3377a49036)

This time I implemented only no.1 in the above specification lists using setTimeout to display lists of 5 images after 3 seconds from API

### CODE REVIEW

File: lesson17/main.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+const createListsOfImg = (imgData) => {
+  const fragment = document.createDocumentFragment();
+  imgData.forEach((imgVal,index) => {
+    const li = createElementWithClassName("li","slideshow-img");
+    const img = document.createElement("img");
+    img.src = imgVal.src;
+
+    index === 0 && li.classList.add("is-show");
+
```
Comment (nyattsu72) [inline lesson17/main.js]: [ask]
I learned how to write this code for the first time! What is this being processed?

Comment (yuka830) [reply lesson17/main.js]: This is the [logical and(&&)](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Logical_AND)

In this situation, if the left side `index === 0` is true then the right side `li.classList.add("is-show")` will be executed. If left side is false, it just returns the left side.
It works the same way as below.
```
if (index === 0) {
      li.classList.add("is-show");
    }
```
I think the logical and is easier and more simple way to write than if statement, so I use it here :)
(I also got this advice from other student before!)

**Comment by suefrontend** [comment]
Is Codesandbox working?

File: lesson17/main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+};
+
+const loading = () => {
+  const loader = document.getElementById("js-loader");
+  loader.classList.add("loading");
+};
+
+const loaded = () => {
+  const loader = document.getElementById("js-loader");
+  loader.classList.add("loaded");
```
Comment (suefrontend) [inline lesson17/main.js:27]: I found this code interesting as I just did "removeChild" to hide the loading image. Great work!

Comment (yuka830) [reply lesson17/main.js:27]: Thank you for reviewing.
When I searched making the loading screen before, I learned this way. So I implemented the same way this time.But removing sounds more simple and easy.
Next time I will do the way you taught me.

**Pull request merged**